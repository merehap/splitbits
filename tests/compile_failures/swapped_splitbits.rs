use splitbits::*;

fn main() {
    splitbits!("aaabbccc", 0b11011101);
}
