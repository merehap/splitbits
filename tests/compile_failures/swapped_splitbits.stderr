error: proc macro panicked
 --> tests/compile_failures/swapped_splitbits.rs:4:5
  |
4 |     splitbits!("aaabbccc", 0b11011101);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: The template expression must come after the input value(s), and must be a string literal, but found:
          0b11011101
